devops/azure-pipelines/build-job-template.yml

```diff
@@ -12,5 +12,7 @@ jobs:
         displayName: 'Install Node.js'
       - script: npm ci
         displayName: 'npm install'
+      - script: npm test -- --watchAll=false
+        displayName: 'npm test'
       - script: npm run build
         displayName: 'npm build'
```

Comment:
typically you'd build before you ran tests, since build errors are generally more fundamental than test failures. if you see other people doing it this way for a good reason in the react space, though, go ahead.